urdf_model/include/urdf_model/utils.h

```diff
@@ -0,0 +1,67 @@
+/*********************************************************************
+* Software License Agreement (BSD License)
+*
+*  Copyright (c) 2008, Willow Garage, Inc.
```

Comment:
Out of the curiosity, shouldn't we use the current year for licenses of new files?